EMS/Helpers/Logger.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EmployeeManagement
+{
+    public class Logger:ILogger
+    {
+        public void LogSuccess(string message)
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
```

Comment:
code is repeating. you can create another private function and use that inside all functions.